### PR TITLE
Feature: project-hello-microverse

### PR DESCRIPTION
**Pull request by Vishal30M**
- Created a new GitHub repo.
- Created a new feature branch.
- Added a .gitignore file.
- Added descriptive README file to my project.
- Added 3 files : index.html, style.css, test.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,34 @@
-# HTML & CSS3 Course
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters, read [root level README](../README.md).
+# project-hello-microverse
 
-## Set-up GitHub Actions
+> In this project, I set up a "Hello world" repository. No complex coding was required for this exercise. My goal was to master all of the tools and best practices I learned about in previous steps.
```
Comment (Kossi-stack) [inline README.md:5]: - [ ] kindly change the Hello-word by Hello Microverse

File: README.md [hunk 2]
```diff
@@ -1,97 +1,34 @@
...
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
+- HTML
+- CSS
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+## Authors
 
-Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+👤 **Vishal Mishra**
```
Comment (Kossi-stack) [inline README.md:14]: - [ ] Kindly follow this example to set up your social handle
Capturemedia

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Vishal30M,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions
 - [ ] [OPTIONAL] Kindly edit the PR's title to something more descriptive. it can be "Hello Microverse Project PR or Hello Microverse Project".

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Vishal30M  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**